dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -3279,6 +3341,90 @@ async fn cmd_nexus_blueprints_import(
     Ok(())
 }
 
+async fn cmd_nexus_chicken_switches_show(
+    client: &nexus_client::Client,
+    args: &ChickenSwitchesShowArgs,
+) -> Result<(), anyhow::Error> {
+    let res = match args.version {
+        ChickenSwitchesVersionOrCurrent::Current => {
+            client.reconfigurator_chicken_switches_show_current().await
+        }
+        ChickenSwitchesVersionOrCurrent::Version(version) => {
+            client.reconfigurator_chicken_switches_show(version).await
+        }
+    };
+
+    match res {
+        Ok(switches) => {
+            println!("Reconfigurator Chicken Switches: ");
+            println!("    version: {}", switches.version);
+            println!("    modified time: {}", switches.time_modified);
+            println!("    planner enabled: {}", switches.planner_enabled);
```

Comment:
nit, take it or leave it: when writing code like this, I often like to destructure the type, like this:
```suggestion
        Ok(switches) => {
            let ReconfiguratorChickenSwitches {
                version,
                modified_time,
                planner_enabled,
            } = switches;
            println!("Reconfigurator Chicken Switches: ");
            println!("    version: {version}");
            println!("    modified time: {modified_time}");
            println!("    planner enabled: {planner_enabled}");
```

The nice thing about this is that destructuring like this matches exhaustively, so if new fields are added to the struct, it will result in a compiler error. This serves as a nice reminder to remember to update the code that prints out the values to handle the new field.

Reply:
Good idea! I'll try this out.

Reply:
Done in [7531c64](https://github.com/oxidecomputer/omicron/pull/8449/commits/7531c641fdaae07fb8d9af5b657297cd6f117d1c)